packages/amazonq/src/app/inline/completion.ts

```diff
@@ -246,8 +245,7 @@ export class AmazonQInlineCompletionItemProvider implements InlineCompletionItem
                 return []
             }
 
-            const start = document.validatePosition(editor.selection.active)
```

Comment:
In non-testing scenarios, `editor.selection.active` appears to always be the same as the position passed in here.